content/00_preface/introduction.adoc

```diff
@@ -101,6 +101,36 @@ The defined look-up tables include:
 
 Examples for _material property_ files and look-up table files are provided in the https://github.com/asam-ev/OpenMATERIAL-3D/tree/main/examples/[examples folder] of the repository.
 
+== Interaction with other ASAM standards
+
+The ASAM OpenX standards strive to provide comprehensive, realistic, and exchangeable simulation data for the development and testing of autonomous systems.
+3D objects defined in the 07_geometry/geometry-index.adoc[Geometry] chapter act as a connecting link between {THIS_STANDARD} and other ASAM standards:
```

Comment:
Technically, the link is not really the 3D object, but the OpenMATERIAL 3D asset file. Because the .xoma file is referenced in the other standards and not the gltf, fbx or usd file.